solr/core/src/test/org/apache/solr/handler/ReplicationTestHelper.java

```diff
@@ -202,15 +199,16 @@ public static NamedList<Object> getDetails(SolrClient s) throws Exception {
     @SuppressWarnings("unchecked")
     NamedList<Object> details = (NamedList<Object>) res.get("details");
 
-    assertNotNull("null details", details);
+    SolrTestCaseJ4.assertNotNull("null details", details);
 
     return details;
   }
 
   public static void assertReplicationResponseSucceeded(NamedList<?> response) {
-    assertNotNull("null response from server", response);
-    assertNotNull("Expected replication response to have 'status' field", response.get("status"));
-    assertEquals("OK", response.get("status"));
+    SolrTestCaseJ4.assertNotNull("null response from server", response);
```

Comment:
I'll take a look to see what can be done here.

Reply:
So the way to fix this is to go back to static import for this. I have no strong opinion either way on this. I'll leave it up to @mkhludnev and @HoustonPutman since I think both of you asked about it.

Reply:
let's keep your option @risdenk